cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -67,7 +67,7 @@ func GetStaticPodSpecs(cfg *kubeadmapi.InitConfiguration, k8sVersion *version.Ve
 			ImagePullPolicy: v1.PullIfNotPresent,
 			Command:         getControllerManagerCommand(cfg, k8sVersion),
 			VolumeMounts:    staticpodutil.VolumeMountMapToSlice(mounts.GetVolumeMounts(kubeadmconstants.KubeControllerManager)),
-			LivenessProbe:   staticpodutil.ComponentProbe(cfg, kubeadmconstants.KubeControllerManager, 10252, "/healthz", v1.URISchemeHTTP),
+			LivenessProbe:   staticpodutil.ComponentProbe(cfg, kubeadmconstants.KubeControllerManager, 10257, "/healthz", v1.URISchemeHTTPS),
```

Comment:
From the documentation https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/ it seems that when using HTTPS probes with hostnetwork=true, it is necessary to set the host header.
Is this the case for kubeadm as well or not?